<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="8.03s"
		durationms="8029"
		type="root"
		rid="20456a73e1696133"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="densenv04"
		rxversion="9.3.4.0"
		host="DESENV04"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="pt-BR"
		dtnlanguage="pt-BR"
		screenresolution="2966x900"
		timestamp="29/12/2020 17:24:21"
		timestampiso="2020-12-29T21:24:21"
		endtime="29/12/2020 17:24:29"
		timeoutfactor="1"
		runid=""
		sutversion="">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:07.988"
			timeRelativeToTestModuleStartTime="00:07.988"
			timeWallClock="17:24:29"
			time="00:07.988"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
